packages/node-patches/src/fs.ts

```diff
@@ -0,0 +1,345 @@
+import { Stats, Dirent, Dir } from 'fs';
+import * as path from 'path';
+import * as util from 'util';
+// using require here on purpose so we can override methods with any
+// also even though imports are mutable in typescript the cognitive dissonance is too high because es modules
+const _fs = require('fs');
+
+//tslint:disable-next-line:no-any
+export const patcher = (fs: any, root: string) => {
+  fs = fs || _fs;
```

Comment:
Style question: Should these be default arguments?

Reply:
weird things happen to the types when i pass fs as a default arg and any so i don't think its worth it now